src/rhsm/profile.py

```diff
@@ -22,8 +22,10 @@
 
 try:
     import dnf
+    import libdnf
 except ImportError:
     dnf = None
+    libdnf = None
```

Comment:
You should probably split this into 2 try/except blocks (one for dnf, one for libdnf), because when I installed dnf (but not libdnf), the except block set dnf to None (even though dnf was successfully imported). Unless this is by design, in which case, ignore this.

Also, how can I import the libdnf python module? (pip search doesn't find it)

Reply:
@nikosmoum I'm sorry that I didn't provide more instruction for testing this PR. It could be possible to install libdnf from Fedora repository, but you will need master branch of libdnf.

This PR can be tested on Fedora 28 (support for modularity). Look at this two articles:

1. https://fedoramagazine.org/modularity-fedora-28-server-edition/
2. https://fedoramagazine.org/working-modules-fedora-28/

and add install some modules to the system. You will also need master branch of libdnf and master branch of DNF using master branch of libdnf. It is not necessary to install libdnf to the system. Instruction for our feature branch related to libdnf plugins can help you:

https://github.com/candlepin/subscription-manager/tree/feature/libdnf_plugins/src/dnf-plugins/test_plugin#not-installing-to-the-root

When you will test this PR, then you will have to add path of DNF source code to `PYTHONPATH`.

Reply:
And you are right. I should split importing of two modules into two try/except blocks.